lib/edoc/doc/overview.edoc

```diff
@@ -377,6 +377,16 @@ The following tags can be used before a function definition:
       Useful for debug/test functions, etc. The content can be
       used as a comment; it is ignored by EDoc.</dd>
 
+  <dt><a name="ftag-private">`@returns'</a></dt>
```

Comment:
Not sure if the "name" attributes are actually used, but the others match the tag name (`ftag-hidden`, `ftag-equiv`) so perhaps these should as well, i.e. `ftag-returns` and `ftag-param` instead of `ftag-private`.